about/package-scope.md

```diff
@@ -18,8 +18,6 @@ Currently, the packages that pyOpenSci reviews also need to fall into the
 technical and applied scope of our organization. This scope may expand over time
 as the organization grows.
 
-
-
 ## Is Your Package in Scope For pyOpenSci Review?
 
 pyOpenSci only reviews packages that fall within our specified domain and
```

Comment:
Should we keep the term "categories" for types of packages and instead only talk about "domains" when we mean fields of study (like in the sense "domain expertise") like earth science, genomics, etc?

If we overload the word "domain" then it will might confuse readers when we use it to refer to "field of study" specifically.

I kind of prefer the way rOpenSci uses the term "package categories" exactly because it avoids mixing two different meaning of "domain"

So this would be "fall within our specified category and technical scope"

Reply:
definitely agreed. will be more clear with that fix.